selfdrive/ui/ui.c

```diff
@@ -802,6 +803,11 @@ static void ui_draw_vision(UIState *s) {
 static void ui_draw_alerts(UIState *s) {
   const UIScene *scene = &s->scene;
 
+  // dont draw alerts that are outdated by > 20 secs
+  if ((nanos_since_boot() - scene->alert_ts) >= 20000000000ULL) {
```

Comment:
When controlsd exists with an alert active. Alerts should eventually be refactored to be managed by ui instead of controls.